tidy3d/components/data/monitor_data.py

```diff
@@ -286,6 +311,16 @@ def dot(
             and the sum of carried power fractions exceed 1. In the non-conjugated definition,
             orthogonal modes can be defined, but the interpretation of modal overlap as power
             carried by a given mode is no longer valid.
+
+        Note
```

Comment:
it would be nice if we did the `. finite_grid_copy` automatically, if possible without the user needing to worry about it. Is it possible?